stingray/pulse/overlapandsave/ols.py

```diff
@@ -0,0 +1,355 @@
+"""
+The code in this file was copied from
+https://github.com/fasiha/array_range
+https://github.com/fasiha/nextprod-py
+https://github.com/fasiha/overlap_save-py
+
+Thanks to Ahmed Fasih for releasing this to the larger public with the
+Unlicense.
+
+# Fast-convolution via overlap-save: a partial drop-in replacement for scipy.signal.fftconvolve
+
+Features:
```

Comment:
Is the text below just the same text as in the readme above? If so, can we consolidate it into a single place?

Reply:
Done